hosts_test.go

```diff
@@ -3,6 +3,7 @@ package mackerel
 import (
 	"encoding/json"
 	"fmt"
+	"io"
```

Comment:
[note] `io/ioutil` has been deprecated. We should use `io` package including `ReadAll()` method.